### PR TITLE
Docs config vars

### PR DESCRIPTION
**Pull request by eranelbaz**
### Issue & Steps to Reproduce / Feature Request
[//]: <> (choose “resolves” “fixes” or “closes” and put link for the issue)
Docs are unclear
### Solution

Better explanation for resource and data
added `import` 

https://github.com/hashicorp/terraform-plugin-docs

### CODE REVIEW

**Review by avnerenv0 — CHANGES_REQUESTED** [review]
@eranelbaz Like I wrote you - running `./generate-docs.sh` will overwrite all these changes ... so please make these changes the way that github.com/hashicorp/terraform-plugin-docs expects and supports them

File: docs/resources/configuration_variable.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+terraform import env0_configuration_variable.my_config '{  "Scope": "PROJECT",  "ScopeId": "project id",  "name": "configuration variable name"}'
```
Comment (avnerenv0) [inline docs/resources/configuration_variable.sh:1]: this should be under the `examples` folder

Comment (eranelbaz) [reply docs/resources/configuration_variable.sh]: moved 7ab7d3f6807cd035167258b3c0f37cb30dfa4110
renamed 7ab7d3f6807cd035167258b3c0f37cb30dfa4110

**Comment by eranelbaz** [comment]
@avnerenv0 I did rollback my changes
and ran the `generate-docs`

**Review by avnerenv0 — APPROVED** [review]

**Pull request merged**